week-2/Homework/mandatory/2-function-creation.js

```diff
@@ -15,18 +25,30 @@ Complete the function to check if the variable `num` satisfies the following req
 Tip: use logical operators
 */
 
-function validate(num) {}
+function validate(num) {
+
+  if (num === num && num % 2 === 0 && num <= 100) {
```

Comment:
what is num===num?
you want to check if it's a number type right?

Reply:
(typeof num === "number" && num % 2 === 0 && num <= 100)
 where should we hve to usr typeof????????????